Dockerfile-dogstatsd-alpine

```diff
@@ -6,40 +6,40 @@ ENV DD_HOME=/opt/datadog-agent \
     DOCKER_DD_AGENT=yes \
     # prevent the agent from being started after install
     DD_START_AGENT=0 \
-    AGENT_VERSION=5.16.0
-
-# Add install and config files
-ADD https://raw.githubusercontent.com/DataDog/dd-agent/master/packaging/datadog-agent/source/setup_agent.sh /tmp/setup_agent.sh
-COPY entrypoint.sh supervisor.conf /
-
-# Expose supervisor and DogStatsD port
-EXPOSE 9001/tcp 8125/udp
+    AGENT_VERSION=5.16.0 \
+    DD_ETC_ROOT="/opt/datadog-agent/agent" \        
+    PATH="/opt/datadog-agent/venv/bin:/opt/datadog-agent/agent/bin:$PATH" \
+    PATH="${DD_HOME}/venv/bin:${DD_HOME}/bin:${PATH}" \
```

Comment:
Actually, as we set them in the same ENV command, I had issues where variables were not set in the described order ending up in PATH=`/venv/bin:/bin:${PATH}"` after evaluation so I've sticked to the full path and removed the other that I forgot to remove there.
Thx